src/docker/run.sh

```diff
@@ -1,3 +1,11 @@
-#!/bin/sh
+#!/bin/bash
+set -euox pipefail
+IFS=$'\n\t'
 
-exec java -jar /app/target/scala-2.11/FireCloud-Orchestration-assembly-*.jar
+
+if [ -e /app/target/scala-2.11/FireCloud-Orchestration-assembly-*.jar ]; then
+  exec java -jar /app/target/scala-2.11/FireCloud-Orchestration-assembly-*.jar
+fi
+
+cd /app
+exec sbt '~ reStart'
```

Comment:
make this an if-else-fi ? Otherwise ... will we run sbt reStart when java exits? (not sure under what condition that would happen, but still)